### PR TITLE
Accessibility: add aria-hidden (and some aria-label)

### PR DESCRIPTION
**Pull request by yannicka**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | The old PrestaShop theme had a number of accessibility problems. Let's try to do better with this new theme! This PR adds "aria-hidden" or "aria-label" to some elements. There's still a lot to do, but it's at least a start.
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| Sponsor company   | N/A
| How to test?      | Install the theme and test all pages

### CODE REVIEW

File: templates/customer/password-new.tpl [hunk 1]
```diff
@@ -48,13 +48,13 @@
...
-              data-text-hide="{l s='Hide' d='Shop.Theme.Actions'}"
-            >
+          <button
+            class="btn btn-primary"
+            type="button"
+            data-action="show-password"
+            data-text-show="{l s='Show' d='Shop.Theme.Actions'}"
+            data-text-hide="{l s='Hide' d='Shop.Theme.Actions'}"
+          >
             <i class="material-icons">visibility</i>
```
Comment (matthieu-rolland) [inline templates/customer/password-new.tpl:58]: Is there a reason this one for not adding `aria-hidden` on this one ?

**Review by matthieu-rolland — APPROVED** [review]
thank you for your contribution @yannicka :+1: 

I'm not so familiar with `aria-hidden`, very nice idea indeed !

File: templates/customer/password-new.tpl (hunk 1 above)
Comment (Hlavtox) [reply templates/customer/password-new.tpl:58]: @matthieu-rolland It's because the button does not have any text, so the icon actually serves a purpose.

**Review by Hlavtox — APPROVED** [review]

File: templates/customer/password-new.tpl (hunk 1 above)
Comment (yannicka) [reply templates/customer/password-new.tpl:58]: @matthieu-rolland Fixed on PR #508

**Pull request merged**